frontend/.eslintrc.json

```diff
@@ -1,31 +1,53 @@
 {
-    "env": {
-        "browser": true
-    },
-    "extends": [
-        "airbnb-base",
-        // "plugin:vue/base"
-        "plugin:vue/recommended"
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    "airbnb-base",
+    "plugin:vue/recommended",
```

Comment:
Is it okay to uncomment this?

Reply:
Also the other uncommented things, actually can you work on them?

Reply:
@gerhean I did not uncomment those, I just removed them entirely. I think we can add it back now but since it is not in used, I think we can just remove those to avoid any potential future errors.